README.md

```diff
@@ -164,13 +257,15 @@ read [Akka documentation about serialization](https://doc.akka.io/docs/akka/curr
 [`CirceAkkaSerializer`](https://github.com/VirtusLab/akka-serialization-helper/blob/main/circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala) Scaladoc
 and look at the [examples](https://github.com/VirtusLab/akka-serialization-helper/tree/main/examples).
 
-### 4. Codec Registration Checker
+## Missing Codec registration
 
-Compiler plugin for checking whether all codecs are registered.
-During compilation, it gathers all direct descendants of the class marked with `@org.virtuslab.ash.SerializabilityTrait`
-and checks the body of classes annotated with `@org.virtuslab.ash.Serializer` if they reference all these direct descendants in any way.
+If we forget to register a codec we would have problems in run time. To solve that we can use
```

Comment:
`runtime` is typically written as a single world (at least from what I've seen in Scala contexts)